web/instances/instances.go

```diff
@@ -146,10 +146,6 @@ func listHandler(c echo.Context) error {
 
 	objs := make([]jsonapi.Object, len(is))
 	for i, in := range is {
-		in.OAuthSecret = nil
```

Comment:
Leaving the OauthSecret & SessionSecret in the client lets any app do anything...